### PR TITLE
RA - 1077 Fixed major part of code by using codeguide

### PR DESCRIPTION
**Pull request by tmarzeion**

### CODE REVIEW

File: app/js/classAdd.controller.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+angular
+    .module('conceptDictionaryApp')
+    .controller('ClassAdd', ['ClassesService', '$location', 'openmrsRest',
+    function(ClassesService, $location, openmrsRest){
```
Comment (rkorytkowski) [inline app/js/classAdd.controller.js]: Apply https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#style-y024

File: app/js/classAdd.controller.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+angular
```
Comment (rkorytkowski) [inline app/js/classAdd.controller.js:1]: Apply https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#style-y010

File: app/js/conceptLocale.service.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+						getLocaleDescr: getLocaleDescr
+					}
+		};
+		/**
+		 * @descriptions array of descriptions objects of concept
+		 * @names array of names objects of concept
+		 * @serverLocales array of available locales
+		 * @returns array of locales of names and descriptions
+		 */
+		function getLocales (names, descriptions, serverLocales){
```
Comment (rkorytkowski) [inline app/js/conceptLocale.service.js:19]: functions belonging to conceptLocaleService should be declared within conceptLocalService above line 15.

File: app/js/conceptSearch.controller.js [hunk 4]
```diff
@@ -0,0 +1,149 @@
+angular
+    .module('conceptDictionaryApp')
+    .controller('ConceptSearch', ['$scope', '$routeParams' ,'ConceptsService', function($scope, $routeParams, ConceptsService) {
```
Comment (rkorytkowski) [inline app/js/conceptSearch.controller.js]: Apply same styles as in classAdd.controller.js

**Comment by rkorytkowski** [comment]
Good progress. It appears that you forgot to update to the latest master. Please remember to do it.

File: app/js/classAdd.controller.js (hunk 2 above)
Comment (tmarzeion) [inline app/js/classAdd.controller.js:1]: We will wait with this one, because we won't need it when we will use webpack

**Pull request merged**